kairon/shared/llm/processor.py

```
@@ -330,6 +330,23 @@ async def __collection_search__(self, collection_name: Text, vector: List, limit
+                                           raise_err=True):
+        client = AioRestClient()
+        response = await client.request(http_url=urljoin(self.db_url, f"/collections/{collection_name}/points/delete"),
+                                        request_method="POST",
+                                        headers=self.headers,
+                                        request_body={"points": point_ids},
+                                        return_json=True,
+                                        timeout=5)
+        print(response)
+
+        if not response.get('result'):
+            if "status" in response:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)